examples/03_connectivity/plot_inverse_covariance_connectome.py

```diff
@@ -60,16 +60,13 @@
 # Display the connectome matrix
 # ------------------------------
 from matplotlib import pyplot as plt
-
+from nilearn.plotting.matrix_plotting import plot_matrix
```

Comment:
Same comment: the import shouldn't be necessary, we should be able to use "plotting.plot_matrix".